scripts/cybershake/auto_submit.py

```diff
@@ -19,79 +20,112 @@
+                cmd = "python $gmsim/workflow/scripts/install_bb.py --v1d %s" % default_1d_mod
+            print cmd
+            call(cmd, shell=True)
+        hf_cmd = "python $gmsim/workflow/scripts/submit_hf.py --binary --auto --srf %s" % run_name
+
         if hf_seed is not None:
             hf_cmd = "{} --seed {}".format(hf_cmd, hf_seed)
         if rand_reset:
             hf_cmd = "{} --rand_reset".format(hf_cmd)
         print hf_cmd
         call(hf_cmd, shell=True)
-        call("echo 'submitted time: %s' >> %s"%(submitted_time,os.path.join(ch_log_dir,'HF.%s.log'%run_name)), shell=True)
-    if proc_type == 5:
-        print "python $gmsim/workflow/scripts/submit_bb.py --binary --auto --srf %s"%run_name
-        call("python $gmsim/workflow/scripts/submit_bb.py --binary --auto --srf %s"%run_name, shell=True)
-        call("echo 'submitted time: %s' >> %s"%(submitted_time,os.path.join(ch_log_dir,'BB.%s.log'%run_name)), shell=True)
-    if proc_type == 6:
-        #TODO: fix inconsistant naming in sub_imcalc.py
-        tmp_path = os.path.join(mgmt_db_location,'Runs')
-            
-        cmd = "python $gmsim/workflow/scripts/submit_imcalc.py --auto --sim_dir %s --i %s --mgmt_db %s --simple_output"%(tmp_path,run_name, mgmt_db_location)
+
+    if proc_type == Process.BB.value:
+        cmd = "python $gmsim/workflow/scripts/submit_bb.py --binary --auto --srf %s" % run_name
+        print cmd
+        call(cmd, shell=True)
+
+    if proc_type == Process.IM_calculation.value:
+        # TODO: fix inconsistent naming in sub_imcalc.py
+        tmp_path = os.path.join(mgmt_db_location, 'Runs')
+        cmd = "python $gmsim/workflow/scripts/submit_imcalc.py --auto -s --sim_dir %s --i %s --mgmt_db %s" % (tmp_path, run_name, mgmt_db_location)
         if extended_period == True:
             cmd = cmd + ' -e'
         print cmd
         call(cmd, shell=True)
-        #save the job meta data
-        call("echo 'submitted time: %s' >> %s"%(submitted_time,os.path.join(ch_log_dir,'IM_calc.%s.log'%run_name)), shell=True)
+
+    fault = run_name.split('_')[0]
+    source_path = os.path.join(mgmt_db_location, 'Data/Sources', fault, 'srf', run_name)
+    srf_path = source_path + '.srf'
+
+    if do_verification:
+        if proc_type == Process.rrup.value:
+            tmp_path = os.path.join(mgmt_db_location, 'Runs')
+            rrup_dir = os.path.join(mgmt_db_location, 'Runs', fault, 'verification')
+            cmd = "python $gmsim/workflow/scripts/submit_imcalc.py --auto -s --sim_dir %s --i %s --mgmt_db %s -srf %s -o %s" % (tmp_path, run_name, mgmt_db_location, srf_path, rrup_dir)
+            print cmd
+            call(cmd, shell=True)
+
+        if proc_type == Process.Empirical.value:
+            cmd = "$gmsim/workflow/scripts/submit_empirical.py -np 40 -i {} {}".format(run_name, mgmt_db_location)
+            print cmd
+            call(cmd, shell=True)
+
+        if proc_type == Process.Verification.value:
+            pass
```

Comment:
I assume this is meant to do nothing for now?

Reply:
Exactly.